Bottom: 10603a9a6234c0df2c77d2bdc3247b6b8925eead
Top:    b8af977582943fe8e41b6eb11a57336280657ed1
Author: Ian Kent <raven@themaw.net>
Date:   2013-02-25 14:24:39 +0800

autofs-5.0.7 - depricate nosymlink pseudo option

The undocumented "nosymlink" option was the only way to force local
NFS mounting until the more descriptive "nobind" option was added.

So depricate the "nosymlink" option in favour of the "nobind" option.


---

diff --git a/CHANGELOG b/CHANGELOG
index 7cf4847..7939daf 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -30,6 +30,7 @@
 - workaround missing GNU versionsort extension.
 - fix wildcard multi map regression.
 - fix file descriptor leak when reloading the daemon.
+- depricate nosymlink pseudo option.
 
 25/07/2012 autofs-5.0.7
 =======================
diff --git a/modules/mount_nfs.c b/modules/mount_nfs.c
index bbbb1de..e61320b 100644
--- a/modules/mount_nfs.c
+++ b/modules/mount_nfs.c
@@ -125,6 +125,10 @@ int mount_mount(struct autofs_point *ap, const char *root, const char *name, int
 
 			o_len = end - cp + 1;
 			if (strncmp("nosymlink", cp, o_len) == 0) {
+				warn(ap->logopt, MODPREFIX
+				     "the \"nosymlink\" option is depricated "
+				     "and will soon be removed, "
+				     "use the \"nobind\" option instead");
 				nosymlink = 1;
 			} else if (strncmp("nobind", cp, o_len) == 0) {
 				nobind = 1;
